src/encoded/static/components/experiment.js

```diff
@@ -428,13 +428,23 @@ const ExperimentComponent = ({ context, auditIndicators, auditDetail }, reactCon
         });
     }
 
-    // Collect biosample docs.
+    // Collect biosample docs
+    // Collect plasmid maps from applied_modifictions documents.
     let biosampleDocs = [];
+    let plasmidMapDocs = [];
     biosamples.forEach((biosample) => {
         biosampleDocs = biosampleDocs.concat(CollectBiosampleDocs(biosample));
         if (biosample.part_of) {
             biosampleDocs = biosampleDocs.concat(CollectBiosampleDocs(biosample.part_of));
         }
+        if (biosample.applied_modifications && biosample.applied_modifications.length > 0) {
+            biosample.applied_modifications.forEach((am) => {
+                const plasmidMapDoc = (am.documents && am.documents.length > 0) ? am.documents.filter(doc => doc.document_type === 'plasmid map') : [];
+                plasmidMapDocs = plasmidMapDocs.concat(plasmidMapDoc);
+            }
+            );
```

Comment:
The `}` and `);` should be on the same line. Did you install a linting plug-in? Linting should help you catch these things.

Reply:
yes, I have the listing installed and activated. It didn't catch this.